mesmer/mesmer_m/harmonic_model.py

```diff
@@ -76,72 +82,49 @@ def fit_fourier_series_np(x, y, n, repeat=False):
     coeffs : array-like of shape (4*n-2)
         Fitted coefficients of Fourier series.
 
-    y : array-like of shape (n_samples*12,)
+    preds : array-like of shape (n_samples*12,)
         Predicted monthly temperature values.
 
     """
 
-    if repeat:
-        # each month scales to the yearly value at that timestep so need to repeat
-        x_train = np.repeat(x, 12)
-        # yearly temp. array for fitting if not already done
-    else:
-        x_train = x
+    # each month scales to the yearly value at that timestep so need to repeat
+    x_train = yearly_predictor #np.repeat(yearly_predictor, 12)
 
     # also get monthly values
     mon_train = np.tile(np.arange(1, 13), int(x_train.shape[0] / 12))
     # for simplicity's sake we take month values in there harmonic form
-    mon_train = (np.pi * (mon_train % 12 + 1)) / 6
-
-    # construct predictor matrix
-
-    # A = np.hstack(
-    #    (
-    #        [
-    #            np.array(
-    #                [
-    #                    x_train * np.sin(i_n * mon_train),
-    #                    np.sin(i_n * mon_train),
-    #                    x_train * np.cos(i_n * mon_train),
-    #                    np.sin(i_n * mon_train),
-    #                ]
-    #            ).T
-    #            for i_n in range(n)
-    #        ]
-    #    )
-    # )
-
-    def fun(x, n, x_train, mon_train, y):
+    #mon_train = (np.pi * (mon_train % 12 + 1)) / 6 # this is double if we have it in generate_fourier_series_np as well
+
+    def fun(coeffs, n, x_train, mon_train, mon_target):
```

Comment:
```suggestion
    def func(coeffs, n, x_train, mon_train, mon_target):
```

I think it's more common to call it func